scripts/lib/encode.js

```diff
@@ -85,3 +129,13 @@ window.Encode.CharCode = {
     }).join('')
   }
 }
+
+window.Encode.Escape = {
+  hex: value => {
+    return ([...value]).map(c => c.charCodeAt() > 0xff ? c : "\\x" + c.charCodeAt().toString(16)).join('')
+  },
+
+  oct: value => {
+    return ([...value]).map(c => c.charCodeAt() > 0xff ? c : "\\" + c.charCodeAt().toString(8)).join('')
```

Comment:
I would like to use single-quote rather than double-quote for consistency.

i.e. `"\\x"` to `'\\x'`

Reply:
Sure!